catalog/app/containers/NavBar/NavBar.tsx

```diff
@@ -589,7 +589,7 @@ export function NavBar() {
   const classes = useNavBarStyles()
   return (
     <Container>
-      {cfg.disableNavigator || (cfg.alwaysRequiresAuth && isSignIn) ? (
+      {cfg.disableNavigator || (cfg.alwaysRequiresAuth && !authenticated) ? (
```

Comment:
what's the actual purpose of this change? i'm not quite sure it's in line with the original intent

Reply:
If authentication is required, but the user is not authenticated, then we show spacer instead of controls.
So, the purpose is the same as written with code

I interpreted the current state as a bug: we show search and bucket selector on the signup page. These controls don't work for unauthenticated users.